src/engine/strat_engine/pool.rs

```diff
@@ -168,6 +168,21 @@ impl StratPool {
             .save_state(&now().to_timespec(), data.as_bytes())
     }
 
+    /// Expand the physical space allocated to a pool by the DATA_EXPANSION_PERCENT
+    fn extend_data(&mut self, current_size: Sectors) -> EngineResult<Sectors> {
+        let expand_size: Sectors =
+            Sectors((current_size.0 as f64 * (DATA_EXPANSION_PERCENT as f64 / 100.0 as f64))
+                        .round() as u64);
+        if let Some(new_data_regions) = self.block_devs.alloc_space(expand_size) {
+            let dm = DM::new().expect("Could not get DM handle");
```

Comment:
It is fine.  I was thinking I should wait until it can pass the CI.  It needs the new version of device-mapper-rs.

Not sure I follow your comment.  try! for getting the DM handle or alloc_space()?  alloc_space() returns an Option, right?